lib/i18n/tasks/scanners/ruby_ast_scanner.rb

```diff
@@ -132,6 +133,10 @@ def setup_matchers
             scanner: self
           )
         end
+        if Array(config[:extra_matchers]).include?('rails_model')
```

Comment:
Perhaps the config should be a list of class names? This way it is extensible outside of i18n-tasks

Reply:
Hm, yes that makes sense. Like the scanners works?

Reply:
Yeah

Reply:
Done 🙂